changes/1983-vgerak.md

```diff
@@ -0,0 +1,2 @@
+Make host optional in `AnyUrl` to conform to [RFC 8089](https://tools.ietf.org/html/rfc8089#section-2)
+(now allows URLs like `"file:///foo/bar"`).
```

Comment:
Add something about the new type `FileUrl` maybe?

Reply:
Changed it completely, as it was misleading (not reflecting the "new" implementation).